app/src/lib/git/core.ts

```diff
@@ -37,6 +37,9 @@ export interface IGitExecutionOptions extends DugiteExecutionOptions {
 
   /** Should it track & report LFS progress? */
   readonly trackLFSProgress?: boolean
+
+  /** Pass stdin to executed command */
+  readonly stdin?: string | Buffer
```

Comment:
```suggestion
```
`IGitExecutionOptions` extends `DugiteExecutionOptions` which already has the `stdin` so this line is not necessary